versioned/storage/common-tests/src/main/java/org/projectnessie/versioned/storage/commontests/AbstractIndexesLogicTests.java

```diff
@@ -338,4 +341,68 @@ public void perCommitOps() {
 
     soft.assertThat(indexesLogic.commitOperations(commit)).containsExactly(add1, add2, remove1);
   }
+
+  @Test
+  void completeIndexesInCommitChainWithStripes() throws Exception {
```

Comment:
You can get a `Persist` instance with tiny limits, see `TestIndexesLogicImpl.completeIndexesInCommitChain`. A long constant (see below) that fits maybe 3 or 4 times into the incremental index and maybe 5 commits, each adding 2 or 3 keys, should then be enough to trigger the spill-out and reference-index splitting.

Would be good to add an assertion that there are reference index splits.